lib/Handler.js

```diff
@@ -1,15 +1,37 @@
+let isColdStart = true;
+
 /**
  * Provides common functionality for lambda handlers.
  */
 class Handler {
+  /**
+   * A utility method which always returns true.
+   *
+   * @returns Boolean
+   *   Always returns true.
+   */
+  static always() {
+    return true;
+  }
+
+  /**
+   * A utility method which always returns false.
+   *
+   * @returns Boolean
+   *   Always returns false.
+   */
+  static never() {
+    return false;
+  }
+
   /**
    * @type {Object}
    *   The default options for the constructor.
    * @static
    */
   static get defaultOptions() {
     return {
-      enableProfiling: false,
+      shouldProfile: this.never,
```

Comment:
I think this should be `Handler.never`

Reply:
That would prevent implementing classes from extending `never` since it ignores any implementation in the current class and goes straight for the `Handler` version of it. In this case, I suppose that might be appropriate since it is highly unlikely anyone would want to alter the behavior of it. Still, I see no reason to lock it down. As it is, it still points correctly to the the static method since this reference is inside a static method itself.

Reply:
I just don't think that this works in v8. I don't think there even is a `this` inside a static method

Reply:
I think it does work. I've confirmed it in the repl and by manually testing with an existing implementation.

Reply:
Yeah, thanks for clearing this up for me!